src/rcpp_constants.cpp

```diff
@@ -839,6 +839,14 @@ String LAND_CFLUX() {
 return D_LAND_CFLUX;
 }
 
+// An alias for LAND_CFLUX that may be more familiar to carbon cycle people
+//' @rdname carboncycle
+//' @export
+// [[Rcpp::export]]
+String NBP() {
+return D_LAND_CFLUX;
+}
+
```

Comment:
I am slightly confused by the NBP terminology. So NBP = Land_C_Flux but this term is more familiar to the carbon cycle community. However because Hector has multiple biomes or will have multiple biomes does NBP reflect the global land C flux or is it biome specific? Are we opposed to spelling out net biome productivity here in the comments? Am I correct in understanding that both NBP() and LAND_CFLUX() will work as functions in R and return the same string?

Reply:
>Am I correct in understanding that both NBP() and LAND_CFLUX() will work as functions in R and return the same string?

Yes as far as I see. C cycle terminology:
* NEE: exchange between land and atmosphere measured in a single place, negative = uptake
* NECB: full carbon balance, so NEE + a bunch of other things not measured by towers
* NBP: carbon balance over a big area, so including things like fire and LUC

Canonical reference is [Chapin et al. 2006](http://dx.doi.org/10.1007/s10021-005-0105-7).